app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
Perform unpacking here to avoid indexing and make the code more descriptive
```suggestion
    command_name, file_in, file_out, *_= command.split()
```